ksp_plugin/interface.cpp

```diff
@@ -180,8 +180,8 @@ serialization::OblateBody::Geopotential MakeGeopotential(
   }
 
   serialization::OblateBody::Geopotential result;
-  for (auto const& pair : rows) {
-    *result.add_row() = pair.second;
+  for (auto const& [degree, row] : rows) {
```

Comment:
I am wondering if we should write:
```
for (auto const& [_, row] : rows) {
```
in the case where one of the elements is not used.  That would maybe make the code more readable (otherwise I am left wondering where `degree` is used).